Core/CoreTiming.cpp

```diff
@@ -719,10 +522,10 @@ void DoState(PointerWrap &p) {
 
 	if (s >= 3) {
 		DoLinkedList<BaseEvent, GetNewEvent, FreeEvent, Event_DoState>(p, first, (Event **) NULL);
-		DoLinkedList<BaseEvent, GetNewTsEvent, FreeTsEvent, Event_DoState>(p, tsFirst, &tsLast);
+		DoIgnoreUnusedLinkedList(p);
 	} else {
 		DoLinkedList<BaseEvent, GetNewEvent, FreeEvent, Event_DoStateOld>(p, first, (Event **) NULL);
-		DoLinkedList<BaseEvent, GetNewTsEvent, FreeTsEvent, Event_DoStateOld>(p, tsFirst, &tsLast);
+		DoIgnoreUnusedLinkedList(p);
```

Comment:
I think a comment that this is for BC would be good.  Someone will come along and "optimize" this code to remove the do-nothing call otherwise.

-[Unknown]

Reply:
Good point, adding.